easybuild/easyblocks/s/suitesparse.py

```diff
@@ -78,6 +79,15 @@ def configure_step(self):
         self.cfg.update('buildopts', 'BLAS="%s"' % os.getenv('LIBBLAS_MT'))
         self.cfg.update('buildopts', 'LAPACK="%s"' % os.getenv('LIBLAPACK_MT'))
 
+        # Get CUDA and set it up appropriately
+        cuda = get_software_root('CUDA')
+        if cuda:
+            cuda_cc_space_sep = self.cfg.template_values['cuda_cc_space_sep'].replace('.', '').split()
```

Comment:
better to use `self.cfg.get_cuda_cc_template_value('cuda_compute_capabilities')` here to return a user-friendly error in case `cuda_compute_capabilities` is not defined.